docker/reporter/src/main/java/org/arquillian/cube/docker/impl/client/reporter/DockerLogSection.java

```diff
@@ -0,0 +1,22 @@
+package org.arquillian.cube.docker.impl.client.reporter;
+
+import org.arquillian.reporter.api.event.SectionEvent;
+import org.arquillian.reporter.api.event.TestSuiteSection;
+import org.arquillian.reporter.api.model.report.BasicReport;
+
+public class DockerLogSection extends SectionEvent<DockerLogSection, BasicReport, TestSuiteSection> {
+
+    public DockerLogSection() {
+        super("DockerLog");
+    }
+
+    @Override
+    public TestSuiteSection getParentSectionThisSectionBelongsTo() {
+        return new TestSuiteSection();
```

Comment:
How would this TestSuiteSection will work if we run multiple test suites in parallel if it has same id. @MatousJobanek